Lib/socket.py

```diff
@@ -67,6 +67,14 @@
            "has_dualstack_ipv6", "AddressFamily", "SocketKind"]
 __all__.extend(os._get_exports_list(_socket))
 
+def __getattr__(name):
+    if name == "SocketType":
+        import warnings
+        warnings._deprecated(name, remove=(3, 16))
```

Comment:
We don't need to emit the warning here anymore, since `_socket` does it as well.

Reply:
I see that we reviewed the PR with the same idea and at the same time :')